### ISSUE TITLE
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined

### ISSUE THREAD

**Issue by rrousselGit** [antfu/sponsorkit#14]
Hello!

I've tried running the example and ended up with the following error when running the CLI:

```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at new NodeError (node:internal/errors:363:5)
    at validateString (node:internal/validators:119:11)
    at Object.resolve (node:path:1102:7)
    at run (/Users/remirousselet/dev/rrousselgit/freezed/node_modules/sponsorkit/dist/cli.cjs:34:20)
    at async Object.handler (/Users/remirousselet/dev/rrousselgit/freezed/node_modules/sponsorkit/dist/cli.cjs:132:3) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```

Diving into the sources, the problem appears to be the `loadConfig` function, and how `loadEnv` sets `outputDir` to `undefined`, which ends-up overriding the default config

TL;DR what happens is:

```dart
const defaultConfig = { outputDir: 'Hello' }
const env = { outputDir: undefined }

function loadConfig() {
  return {
    ...defaultConfig,
    ...env
  }
}


console.log(loadConfig()) // { outputDir: undefined }
```
